catalog/admin/admin-poller-service/pom.xml

```diff
@@ -59,6 +59,30 @@
             <groupId>ddf.action.core</groupId>
             <artifactId>action-core-api</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Well, yes and know. The 1.4 version of the gmavenplus plugin is in the root pom but it is only going to run if the `findbugs` profile is on. So during a full build it will be invoked and all will be good, but if someone were to want to run tests with findbugs turned off (which, honestly, they shouldn't but...) it would not be run.

Go ahead and add the 1.5 version to this module for now. We may consider doing a bit of Spock cleanup (maybe going so far as backporting the shaded jar) in 2.9.x after the next release but not now.